scripts/util.ts

```diff
@@ -26,59 +31,69 @@ export async function getOwl(
 }
 
 async function setAllowance(
-  token: Truffle.ContractInstance,
+  token: TokenOwlInstance | Erc20MintableInstance,
```

Comment:
Nice! Its cool that it "just works" because they both have an `approve` method with the same signature.